motioncorr/convert.py

```diff
@@ -58,3 +238,43 @@ def getMovieFileName(movie):
         fn += ':ems'
 
     return fn
+
+
+def parseEERDefects(fn):
```

Comment:
So far, relion extracts defects from the gain file differently - 0 pixels mean a defect. Here I extract the same info from the header. It might change in the future (as always with TFS). But I think any plugin that does motion cor on EPU EER data will potentially benefit from this.